### PR TITLE
[mgt] msal domainHint and prompt features 

### PR DESCRIPTION
**Pull request by vogtn**
mgt msal 

ref:
https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/998

### CODE REVIEW

File: concepts/toolkit/providers/msal.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Initializing the MSAL provider in HTML is the simplest way to create a new provi
                    login-type="redirect/popup"
                    scopes="user.read,people.read"
                    redirect-uri="https://my.redirect/uri"
-                   authority=""></mgt-msal-provider>
+                   authority=""
+                   domainHint="mydomain.com"
+                   prompt="consent"></mgt-msal-provider>
```
Comment (nmetulev) [inline concepts/toolkit/providers/msal.md:29]: Please also add these in the config object below under `Initialize in Javascript`

File: concepts/toolkit/providers/msal.md [hunk 2]
```diff
@@ -34,7 +36,9 @@ Initializing the MSAL provider in HTML is the simplest way to create a new provi
...
 | authority    | Authority string - default is the common authority. For single-tenant apps, use your tenant ID or tenant name. For example, `https://login.microsoftonline.com/[your-tenant-name].onmicrosoft.com` or `https://login.microsoftonline.com/[your-tenant-id]`. Optional. |
 | redirect-uri | Redirect URI string - by default the current window URI is used. Optional.                                                                                                                                                                                            |
-| depends-on   | Element selector string of another higher priority provider component. Optional.                                                                                                                                                                                      |
+| depends-on   | Element selector string of another higher priority provider component. Optional. 
+| domain-hint  | Query string of domain location for forwarding sign in experience. Optional.
```
Comment (nmetulev) [inline concepts/toolkit/providers/msal.md:40]: Can we link to docs for this somewhere?

Comment (vogtn) [reply concepts/toolkit/providers/msal.md:40]: I've been using this as a reference:
https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow#send-the-sign-in-request

Comment (vogtn) [reply concepts/toolkit/providers/msal.md:40]: I can replace the dead link on our page here:
https://github.com/microsoftgraph/microsoft-graph-docs/blob/ad3717f659de9779f82805a2201db908d33975d9/concepts/toolkit/providers/msal.md

Under "To learn more about MSAL.js and for additional options you can use when initializing the MSAL library, see the MSAL documentation."

File: concepts/toolkit/providers/msal.md [hunk 3]
```diff
@@ -62,6 +68,8 @@ interface MsalConfig {
   loginType?: LoginType; // LoginType.Popup or LoginType.Redirect (redirect is default)
   loginHint?: string
   options?: Configuration; // msal js Configuration object
+  domainHint?: string;
+  prompt?: string;
```
Comment (nmetulev) [inline concepts/toolkit/providers/msal.md:72]:
```suggestion
  prompt?: string; // "login", "none", "select_account", "consent"
```

File: concepts/toolkit/providers/msal.md (hunk 2 above)
Comment (nmetulev) [reply concepts/toolkit/providers/msal.md:40]: Wow, good catch on the dead link - we should fix that but I don't think the docs you linked are the right ones, looks like these are the replacement: https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-js-initializing-client-applications

Could you add the docs for the prompt option in the description?

File: concepts/toolkit/providers/msal.md (hunk 1 above)
Comment (vogtn) [reply concepts/toolkit/providers/msal.md:29]: updated: 1a77647

File: concepts/toolkit/providers/msal.md (hunk 2 above)
Comment (vogtn) [reply concepts/toolkit/providers/msal.md:40]: updated the link and added the prompt docs:
39fcb00

File: concepts/toolkit/providers/msal.md [hunk 4]
```diff
@@ -34,7 +36,11 @@ Initializing the MSAL provider in HTML is the simplest way to create a new provi
...
-| depends-on   | Element selector string of another higher priority provider component. Optional.                                                                                                                                                                                      |
+| depends-on   | Element selector string of another higher priority provider component. Optional. 
+| domain-hint  | Query string of domain location for forwarding sign in experience. Optional.              
+| prompt | Selection for type of user interaction requited to login. Valid options include: `login` forces the user to enter credentials on request. `none` for no interactive prompt. `select_account` to send the user to an account picker. `consent` to send the user to a OAuth consent dialog. For more prompt information, see the [docs](/azure/active-directory/develop/msal-js-prompt-behavior). Optional.                                                                                                                                                                            |
+
+To learn more about the MSAL specification and optional
```
Comment (nmetulev) [inline concepts/toolkit/providers/msal.md:43]: I think you might have not committed all your changes - the sentence is not complete

Comment (vogtn) [reply concepts/toolkit/providers/msal.md]: Oh sorry... I didn't remove it, I think inside the description should be good enough? Unless you want more.

**Review by nmetulev — APPROVED** [review]

**Pull request merged**